### PR TITLE
Revise “Logging Architecture” topic

### PR DESCRIPTION
**Pull request by sftim**
Revise [Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) [[preview](https://deploy-preview-33876--kubernetes-io-main-staging.netlify.app/docs/concepts/cluster-administration/logging/)]

- Drop mention of Docker
- Mention more about log rotation (there's room for even more)

/sig node

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (zacharysarah) [inline content/en/docs/concepts/cluster-administration/logging.md:144]: Use [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
```suggestion
Kubernetes does not manage log rotation. If you deploy Kubernetes cluster components (such as the scheduler) to log to
a volume shared from the parent node, make sure that those
logs are rotated. 

Your operating system may automatically implement some log rotation. 
For example, if you share the directory `/var/log` into a static Pod for a component, 
node-level log rotation treats a file in that directory the same as a file written
by any component outside Kubernetes.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@sftim Good stuff. ✨ One edit for clarity, otherwise LGTM

**Comment by zacharysarah** [comment]
This looks good apart from one small editorial change, so I'm approving this PR in order to keep it moving. Other reviewers, feel free to LGTM once feedback is addressed.

/sig instrumentation
@kubernetes/sig-instrumentation-approvers 👋🏻 PTAL

/approve

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (sftim) [reply content/en/docs/concepts/cluster-administration/logging.md:144]: You know what, CRIContainerLogRotation is GA so I think this needs a bigger change.

**Comment by sftim** [comment]
I want to fix the `CRIContainerLogRotation` aspect before I mark this as ready for review.

**Comment by sftim** [comment]
This is once again ready for review.

[Relevant changes](https://github.com/kubernetes/website/compare/e9cb3664eaa73d5ddc3ab17aecc14c98ef887cf8..2fbedf110fa8ce2741a0169bb55c4ab8c54d88a4) since last LGTM.

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (reylejano) [inline content/en/docs/concepts/cluster-administration/logging.md:349]: wdyt?
```suggestion
* Learn how [customise the termination message](/docs/tasks/debug/debug-application/determine-reason-pod-failure/#customizing-the-termination-message) that Kubernetes records when a Pod fails
```

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (sftim) [reply content/en/docs/concepts/cluster-administration/logging.md]: I think that's sorted, now.

**Comment by sftim** [comment]
I rebased against _main_.

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/logging.md:349]:
```suggestion
* Learn how to [customize the termination message](/docs/tasks/debug/debug-application/determine-reason-pod-failure/#customizing-the-termination-message) that Kubernetes records when a Pod fails
```

File: content/en/docs/concepts/cluster-administration/logging.md
Comment (kbhawkey) [inline content/en/docs/concepts/cluster-administration/logging.md:343]:
```suggestion
Cluster logging that exposes or pushes logs directly from every application is outside the scope of Kubernetes.
```

**Pull request merged**
